test/functional/cmdLineTests/runtimemxbeanTests/src/GetPid.java

```diff
@@ -24,11 +25,17 @@
 import java.lang.management.ManagementFactory;
 
 class GetPid {
-
-	public static void main(String[] args) {
-		RuntimeMXBean mb = ManagementFactory.getRuntimeMXBean();
-		long id = mb.getPid();
-		System.out.println(id);
+	public static void main(String args[]) {
+		try {
+			RuntimeMXBean mb = ManagementFactory.getRuntimeMXBean();
+			long id = mb.getPid();
+			System.out.println(id);
+			/*
+			 * Puts Java process on hold until system receives input from
+			 * testing script getPidTest.pl
+			 */
+			System.in.read();
+		} catch (Exception e) {
```

Comment:
Better to put the try catch just around the System.in.read() so exceptions are ignored only for this call.